shared/actions/favorite.js

```diff
@@ -164,6 +164,11 @@ const jsonToFolders = (json: Object, myKID: any) => {
 
 export function favoriteList (): (dispatch: Dispatch, getState: () => Object) => void {
   return (dispatch, getState) => {
+    // Ignore any messages we receive while reloading
```

Comment:
maybe we should just print a msg if an action happens while the engine is resetting?